hummingbot/strategy_v2/backtesting/executors_simulator/position_executor_simulator.py

```diff
@@ -30,6 +38,7 @@ def simulate(self, df: pd.DataFrame, config: PositionExecutorConfig, trade_cost:
         df_filtered['cum_fees_quote'] = 0.0
         df_filtered['filled_amount_quote'] = 0.0
         df_filtered["current_position_average_price"] = float(config.entry_price)
+        df_filtered['sl'] = float(sl)
```

Comment:
to avoid confusion with stop loss that usually is sl, I recommend calling it ts (trailing stop)